src/ape/plugins/__init__.py

```diff
@@ -168,7 +168,11 @@ def _register_plugins(self):
         if self.__registered:
             return
 
-        plugins = [x.name.replace("-", "_") for x in distributions() if x.name.startswith("ape-")]
+        plugins = [
+            getattr(x, "name", "").replace("-", "_")
```

Comment:
might be a little clearer, since we know it won't execute this unless `x` has `.name`
```suggestion
            x.name.replace("-", "_")
```

Reply:
Made the change, but PyCharm still highlights it for some reason, throws me off.

<img width="566" alt="Screenshot 2024-04-16 at 11 14 24" src="https://github.com/ApeWorX/ape/assets/19540978/21bd4954-c3e1-45d4-ace4-e2c3b1e5b302">